### PR TITLE
Awais786/ecom 7523 remove fields

### PR DESCRIPTION
**Pull request by awais786**
From course-run-detail `all` tab removed the subjects, image and pre-requisites.
From edit page remove the course fields.

screencapture-127-0-0-1-8009-publisher-course_runs-55-edit-1490865233363

### CODE REVIEW

**Comment by awais786** [comment]
@tasawernawaz Please review.

**Comment by awais786** [comment]
@tasawernawaz @waheedahmed please review.

File: course_discovery/templates/publisher/course_run/edit_run_form.html [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+                                </label> {{ seat_form.type }}
+                            </div>
+                            <div id="SeatPriceBlock" class="col col-6 {% if not seat_form.price.value %}hidden{% endif %}">
+                                <label class="field-label ">{{ seat_form.price.label_tag }}</label>
+                                {{ seat_form.price }}
+                            </div>
+                        </div>
+                        {% if seat_form.price.errors %}
+                            <div class="field-message has-error">
+                                        <span class="field-message-content">
```
Comment (waheedahmed) [inline course_discovery/templates/publisher/course_run/edit_run_form.html]: nit: fix indentation.

File: course_discovery/templates/publisher/course_run/edit_run_form.html [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+                    <div class="col col-6">
+                        <label class="field-label ">{{ run_form.language.label_tag }}</label>
+                        {{ run_form.language}}
+
+                        <label class="field-label ">{{ run_form.transcript_languages.label_tag }}</label>
+                        {{ run_form.transcript_languages}}
+
+                        <label class="field-label margin-top20">{{ run_form.video_language.label_tag }}</label>
+                        {{ run_form.video_language}}
+
```
Comment (waheedahmed) [inline course_discovery/templates/publisher/course_run/edit_run_form.html]: remove this line.

File: course_discovery/templates/publisher/course_run/edit_run_form.html [hunk 3]
```diff
@@ -0,0 +1,282 @@
...
+                            </li>
+                        </ul>
+                    </div>
+                    <div class="col col-6">
+                        <label class="field-label ">{{ run_form.start.label_tag }}  <span class="required">*</span></label>
+                        {{ run_form.start }}
+                    </div>
+                </div>
+
+
```
Comment (waheedahmed) [inline course_discovery/templates/publisher/course_run/edit_run_form.html]: nit: remove extra new line.

File: course_discovery/apps/publisher/views.py [hunk 4]
```diff
@@ -123,7 +123,7 @@ def get_context_data(self, **kwargs):
 class CourseRunDetailView(mixins.LoginRequiredMixin, mixins.PublisherPermissionMixin, DetailView):
     """ Course Run Detail View."""
     model = CourseRun
-    template_name = 'publisher/course_run_detail.html'
+    template_name = 'publisher/course_run_detail/course_run_detail.html'
```
Comment (tasawernawaz) [inline course_discovery/apps/publisher/views.py:126]: move this into `course_run` folder.

**Review by tasawernawaz — APPROVED** [review]
One comment otherwise 👍

File: course_discovery/apps/publisher/views.py (hunk 4 above)
Comment (awais786) [reply course_discovery/apps/publisher/views.py:126]: will move this in cleanup story.

**Pull request merged**